core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -801,6 +818,26 @@ public Build startBinaryBuild(String buildConfigName, File file) {
     }
 
     // BuildConfigs
+
+    // Workaround for NPE:
+    //    java.lang.NullPointerException: unit == null
+    //    at okhttp3.internal.Util.checkDuration(Util.java:496)
+    //    at okhttp3.OkHttpClient$Builder.readTimeout(OkHttpClient.java:596)
+    //    at io.fabric8.openshift.client.dsl.internal.build.BuildConfigOperationsImpl.submitToApiServerWithRequestBody(BuildConfigOperationsImpl.java:283)
+    //    at io.fabric8.openshift.client.dsl.internal.build.BuildConfigOperationsImpl.fromInputStream(BuildConfigOperationsImpl.java:186)
+    //    at io.fabric8.openshift.client.dsl.internal.build.BuildConfigOperationsImpl.fromInputStream(BuildConfigOperationsImpl.java:167)
+    //    at io.fabric8.openshift.client.dsl.internal.build.BuildConfigOperationsImpl.fromInputStream(BuildConfigOperationsImpl.java:72)
+    //    at cz.xtf.core.bm.BinaryBuildFromSources.build(BinaryBuildFromSources.java:60)
+    //    at cz.xtf.core.bm.BuildManager.deploy(BuildManager.java:62)
+    //    at cz.xtf.junit5.listeners.ManagedBuildPrebuilder.testPlanExecutionStarted(ManagedBuildPrebuilder.java:85)
+    //
+    // kubernetes-client issue will be created
```

Comment:
Thanks, could you link the kubernetes-client issue here, please? We might need new XTF issue to remove it once fixed in kubernetes-client so we don't forget it.

Reply:
I need to investigate it more, the code is unchanged. It is not obvious to me now why we need this. I'll create XTF issue to evaluate and investigate the workaround.

Reply:
https://github.com/xtf-cz/xtf/issues/461